near-sdk/src/store/tree_map/iter.rs

```diff
@@ -981,12 +981,26 @@ where
     }
 }
 
+// FIXME: borsh 0.9 didn't feature ser/de implementation for `std::ops::Bound`
+// nor does 1.0.0, so the `BorshDeserialize`, `BorshSerialize` were and are pointless
```

Comment:
How does it even get compiled then? I also don't see a reason for this type fo implement BorshSerialize / BorshDeserialize, do I miss something (would it break if we completely drop them here)?

Reply:
It shouldn't break if we completely drop them here.

the bound is `std::ops::Bound<K>: borsh::ser::BorshSerialize`, so the code compiles.
If an implementation of trait is added for this type in `borsh`, the code will even become runnable.